src/lib/yang/schema.lua

```diff
@@ -795,20 +795,25 @@ function resolve(schema, features)
       end
       if node.body then
          node.body = shallow_copy(node.body)
-         for k,v in pairs(node.body or {}) do
+         local extra = {}
```

Comment:
Apologies for the nitpick here, but I request more changes :)  Instead of `node.body = shallow_copy(node.body)`, can we do `local body = node.body`, and `node.body = {}`.  Then instead of iterating over `node.body`, can we iterate over our local `body` and then add to the new table `node.body`.  WDYT?